packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.js

```diff
@@ -32,6 +32,17 @@ export default class QuanticCategoryFacet extends LightningElement {
   @api engineId;
   /** @type {boolean} */
   @api noSearch = false;
+  /** @type {string} */
+  @api basePath = '';
+  /** @type {boolean} */
+  @api noFilterByBasePath = false;
+  /** @type {string} */
+  @api delimitingCharacter = ';';
+  /** @type {number} */
+  @api numberOfValues = 5;
```

Comment:
is it true that we only put 5 for category facet? as we have 8 for atomic?

Reply:
i changed to 8